src/css/custom.css

```diff
@@ -556,3 +556,35 @@ nav.menu {
     box-shadow: none;
   }
 }
+
+.graphiql-main {
+  height: 80vh;
+}
+
+.graphiql-main * {
+  font-family: 'Space Grotesk', sans-serif !important;
+}
+
+.graphiql-dialog {
+  top: 60% !important;
```

Comment:
avoid !important

Reply:
This is used here to override parent graphiql styles.